router/router.go

```diff
@@ -123,9 +122,9 @@ func New(cfg Config) http.Handler {
 	}
 
 	if cfg.RelCalEnabled {
-		router.Handle(cfg.RelCalRoutePrefix+"/releasecalendar", abtest.Handler(cfg.RelCalEnableABTest, cfg.RelCalHandler, cfg.BabbageHandler, cfg.RelCalABTestPercentage, abtest.RelcalTestCookieAspect, cfg.SiteDomain, abtest.RelcalNewExit))
+		router.Handle(cfg.RelCalRoutePrefix+"/releasecalendar", relcal.Handler(cfg.UseNewReleaseCalendar, cfg.RelCalHandler, cfg.BabbageHandler))
```

Comment:
```suggestion
 if cfg.UseNewReleaseCalendar {
		router.Handle(cfg.RelCalRoutePrefix+"/releasecalendar", relcal.Handler(cfg.RelCalHandler, cfg.BabbageHandler))
} else {
		router.LegacyHandle(cfg.RelCalRoutePrefix+"/releasecalendar", relcal.Handler(cfg.RelCalHandler, cfg.BabbageHandler))
}
```

would prefer the logic to be explicit here, and only done once during setup (i.e. decide now for `Handler` (new) or `LegacyHandler` (old)), rather than during each request